### ISSUE TITLE
2sxc 09.03.01 instllation missing 09.03.01.SqlDataProvider

### ISSUE THREAD

**Issue by d-trattner** [2sic/2sxc#1253]
**I'm submitting a ...**
```
- [x] bug report => search github for a similar issue before submitting 
```

**...about**   
```
- [x] installation
```

**Current behavior**
![grafik](https://user-images.githubusercontent.com/27424165/29207381-4531dd72-7e86-11e7-927f-8b9a796cafed.png)

**Expected behavior**
Installation without error

**Minimal reproduction of the problem with instructions**
Install 2sxc 09.03.01

**What is the motivation / use case for changing the behavior?**
just to inform

**Please tell us about your environment:**
any

* **2sxc version(s):** 9.3.1
* **Browser:** [all]
* **DNN:** [ 9.1.3 ] 
* **Language:** [any/all | English | German | other ]

**Anything you would like to add**

**Comment by iJungleboy** [comment]
thanks - duplicate of #1252 and already fixed. we juts released 09.03.02

**Comment by d-trattner** [comment]
I'm sorry
